.travis.yml

```diff
@@ -13,6 +13,7 @@ branches:
     - develop
     - doc
     - ci
+    - expr_template
```

Comment:
This should not be merged into the develop branch

Reply:
This is outdated. In the last commit named I have removed this. Please check the final changes

Reply:
yes. I now have seen it.